packages/salesforce/src/Adaptor.js

```diff
@@ -104,11 +128,10 @@ export function execute(...operations) {
   };
 
   return state => {
-    util.createConnection(state);
-    connection = util.getConnection();
     return commonExecute(
       util.loadAnyAscii,
-      ...operations
+      createConnection,
+      ...flatten(operations)
```

Comment:
Why did we introduce flatten here? it wasn't here before?

Reply:
@josephjclark, flatten was there before. I just moved `createConnection` from util.js to Adaptor.js and removed `removeConnection` function

Reply:
Ah then I'm getting confused.

Ok, I've looked over this code. You can remove `flatten` completely and close #1067. Let's just do this here, it's a simple change and it's getting merged into an epic anyway

Reply:
@josephjclark i have removed the use of `flatten` and deleted `lodash` package

Reply:
Great, thanks!

Reply:
@josephjclark just a heads up, #1070 is rebased against `687-use-client` branch. So that PR should be merged before this one to avoid conflict resolution